legate/core/communicator.py

```diff
@@ -140,17 +140,13 @@ def needs_barrier(self) -> bool:
         return self._needs_barrier
 
     def _initialize(self, volume: int) -> FutureMap:
-        task = Task(
-            self._context,
-            self._init_cpucoll_id,
-            tag=self._tag,
-            side_effect=True,
-        )
-        cpucoll_id = task.execute_single()
+        cpucoll_uid = self._runtime.core_library.legate_cpucoll_initcomm()
```

Comment:
fyi, technically, this code doesn't work when control replication is turned off, but no communicators would work either.